modules/templates/helper.go

```diff
@@ -286,6 +286,9 @@ func NewFuncMap() []template.FuncMap {
 			}
 			return false
 		},
+		"svg": func(icon string, size int) template.HTML {
+			return template.HTML(fmt.Sprintf(`<svg class="svg %s" width="%d" height="%d"><use xlink:href="%s/img/svg/icons.svg#%s" /></svg>`, icon, size, size, setting.StaticURLPrefix, icon))
```

Comment:
Should we put an alt attr on the image ? (I am only asking because I don't know the exact answer and could be out off scope for this PR)

Reply:
I'm also unsure, but it may be beneficial for a11y? I'm thinking just the name of the icon again?

Reply:
Alt attributes do not work on inline SVG. What does is a `<title>` tag inside `<svg>` but we cut that out using `svgo` to save on size. I don't think it's a big deal for a11y. GitHub's SVG also don't have a title tag.

Reply:
GitHub's SVGs do have `aria-hidden="true"` which may [make sense](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute) because our icons are of cosmetical nature.